Gems/Prefab/PrefabBuilder/PrefabGroup/ProceduralAssetHandler.cpp

```diff
@@ -68,6 +79,84 @@ namespace AZ::Prefab
         extensions.push_back(PrefabGroupAssetHandler::s_Extension);
     }
 
+    void PrefabGroupAssetHandler::AssetTypeInfoHandler::AddContextMenuActions(
+        [[maybe_unused]] QWidget* caller,
+        QMenu* menu,
+        const AZStd::vector<AzToolsFramework::AssetBrowser::AssetBrowserEntry*>& entries)
+    {
+        using namespace AzToolsFramework::AssetBrowser;
+        auto entryIt = AZStd::find_if
+        (
+            entries.begin(),
+            entries.end(),
+            [](const AssetBrowserEntry* entry) -> bool
+            {
+                return entry->GetEntryType() == AssetBrowserEntry::AssetEntryType::Product;
+            }
+        );
+
+        if (entryIt == entries.end())
+        {
+            return;
+        }
+        else  if ((*entryIt)->GetEntryType() == AssetBrowserEntry::AssetEntryType::Product)
+        {
+            ProductAssetBrowserEntry* product = azrtti_cast<ProductAssetBrowserEntry*>(*entryIt);
+            if (product->GetAssetType() == azrtti_typeid<ProceduralPrefabAsset>())
+            {
+                AZ::Data::AssetId assetId = product->GetAssetId();
+                menu->addAction("Save as Prefab...", [assetId, this]()
+                {
+                    QString filePath = AzQtComponents::FileDialog::GetSaveFileName(nullptr, QString("Save to file"), "", QString("Prefab file (*.prefab)"));
```

Comment:
I'm wondering if it would be an idea to have a one time popup (or a window with a "Don't show again" checkbox) that provides some information about the pros/cons of saving a Prefab like this permanently, such as not receiving updates from fbx changes?

Reply:
The normal workflow for starting an authored Prefab is to prompt the user where they want to save the new Prefab, so this prompt is following the same workflow.

Reply:
I think this can be improved over time if customers get confused about it.